addons/point_of_sale/static/src/js/printers.js

```diff
@@ -30,7 +30,7 @@ class PrintResultGenerator {
             successful: false,
             message: {
                 title: _t('Connection to the printer failed'),
-                body: _t('Please check if the printer is still connected. \n' +
+                body: _t('Please check if the printer is still connected and make sure there are no errors in the IoT box logs. \n' +
```

Comment:
Maybe we should rethink this part.
The error here assumes that the printer was not reachable (the request itself failed to be received by the target (Iot or printer) ), in our case, the IoT box was reached AND did replied. Can't we make a distinction as we did receive something to have a more relevant error message ?

I mean, let's assume that a customer receive this error message, it's not obvious what they should do as we give ton of instructions. I think it would really help them if you can have your own custom error message at a more relevant place in the code.